qiling/utils.py

```diff
@@ -376,9 +375,7 @@ def __int_nothrow(v: str, /) -> Optional[int]:
 def select_arch(archtype: QL_ARCH, cputype: Optional[QL_CPU], endian: QL_ENDIAN, thumb: bool) -> QlClassInit['QlArch']:
     kwargs = {}
 
-    # skip cpu model for evm
-    if archtype is not QL_ARCH.EVM:
-        kwargs['cputype'] = cputype
+    kwargs['cputype'] = cputype
```

Comment:
That can be moved to be part of the initialization two lines above, that is:
`kwargs = {'cputype': cputype}`